src/client/pages/OfferDebugger/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { LoadingDots } from 'components/LoadingDots/LoadingDots'
+
+const ActualDebugger = React.lazy(async () => {
+  const module = await import(
+    /* webpackChunkName: "offer-debugger" */ './Debugger'
+  )
+  return { default: module.Debugger }
+})
+
+export const OfferDebugger: React.FC = () => {
+  if (window.hedvigClientConfig.appEnvironment === 'production') {
+    return null
+  }
```

Comment:
I know this is what we do currently with the debugger but it's kind of weird that if you go to this route in prod you just get a black page. Could we redirect to the landing page instead? As we do with all other non-existent routes?

Reply:
Sure, I don't see why not 👍